_rules/button-has-name-97a4e1.md

```diff
@@ -38,10 +38,10 @@ There are no major accessibility support issues known for this rule.
 
 ## Background
 
-- [https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
-- [https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=412#qr-ensure-compat-rsv](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=412#qr-ensure-compat-rsv)
-- [ARIA14: Using aria-label to provide an invisible label where a visible label cannot be used](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html)
-- [https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html](https://www.w3.org/TR/WCAG20-TECHS/ARIA16.html)
+- [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)
+- [How to meet Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value)
```

Comment:
In general this PR seems to replace quickref links with Understanding document links. I think this one could be removed too.